pkg/acme/webhook/registry/challengepayload/challenge_payload.go

```diff
@@ -43,6 +43,8 @@ func NewREST(hookFn webhook.Solver) *REST {
 	}
 }
 
+func (r *REST) Destroy() {}
```

Comment:
Could someone confirm that an empty function is OK here?
Implementing this function is required to fit the changes to the "k8s.io/apiserver/pkg/registry/rest" Storage api.

Reply:
Yep that's okay - this resource type isn't actually persisted anywhere, it is only submitted to the DNS01 solver webhooks, so there's nothing to do to delete a resource/it doesn't make sense in this context. It may be worth adding this as a comment to the empty function for future readers though 😄

Reply:
Ok, thx; I added the comment.